### PR TITLE
Add Ascend and Descend

### PR DESCRIPTION
**Pull request by Fynn93**
Added Ascend and Descend
(Not the Best Implementation but it works!)

### CODE REVIEW

File: src/server/commands/navigation/ascend.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { PlayerUtil } from "@modules/player_util.js";
+import { RawText } from "@notbeer-api";
+import { registerCommand } from "../register_commands.js";
+import { BlockLocation, Location } from "mojang-minecraft";
+
+const registerInformation = {
```
Comment (SIsilicon) [inline src/server/commands/navigation/ascend.ts:6]: The ascend command has a `levels` parameter that determines how many "floors" to go through.
https://worldedit.enginehub.org/en/latest/usage/navigation/?highlight=ascend#ascending-and-descending

File: src/server/commands/navigation/descend.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { PlayerUtil } from "@modules/player_util.js";
+import { RawText } from "@notbeer-api";
+import { registerCommand } from "../register_commands.js";
+import { BlockLocation, Location } from "mojang-minecraft";
+
+const registerInformation = {
+    name: "descend",
```
Comment (SIsilicon) [inline src/server/commands/navigation/descend.ts:7]: The descend command has a `levels` parameter that determines how many "floors" to go through.
https://worldedit.enginehub.org/en/latest/usage/navigation/?highlight=ascend#ascending-and-descending

File: texts/en_US.po [hunk 3]
```diff
@@ -500,6 +500,18 @@ msgid "commands.wedit:thru.none"
 msgstr "Not against or facing a wall to go through."
 msgid "commands.wedit:thru.obstructed"
 msgstr "No free spot ahead of you found."
+msgid "commands.wedit:ascend.description"
+msgstr "Go up a floor"
+msgid "commands.wedit:ascend.explain"
```
Comment (SIsilicon) [inline texts/en_US.po:505]: To optimize the number of language keys used, You can replace `commands.wedit:ascend/descend.explain` with `commands.wedit:thru.explain`.

**Review by SIsilicon — CHANGES_REQUESTED** [review]
If you can add the `levels` parameter to these two commands that would be great.

**Comment by Fynn93** [comment]
Done (see commit #9088179)

**Comment by SIsilicon** [comment]
There seems to be some indentation, trailing spaces and semicolons missing. These are the errors from eslint.
```
/home/runner/work/WorldEdit-BE/WorldEdit-BE/src/library/Minecraft.ts
  194:1  error  Trailing spaces not allowed  no-trailing-spaces

/home/runner/work/WorldEdit-BE/WorldEdit-BE/src/server/commands/navigation/ascend.ts
   7:1   error  Expected indentation of 2 spaces but found 4         indent
   8:1   error  Expected indentation of 2 spaces but found 4         indent
   9:1   error  Expected indentation of 2 spaces but found 4         indent
  [10](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:11):1   error  Expected indentation of 2 spaces but found 4         indent
  [11](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:12):1   error  Expected indentation of 4 spaces but found 8         indent
  [12](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:13):1   error  Expected indentation of 6 spaces but found 12        indent
  [13](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:14):1   error  Expected indentation of 6 spaces but found 12        indent
  [14](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:15):1   error  Expected indentation of 6 spaces but found 12        indent
  [15](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:16):1   error  Expected indentation of 4 spaces but found 8         indent
  [16](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:17):1   error  Expected indentation of 2 spaces but found 4         indent
  [20](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:21):1   error  Expected indentation of 2 spaces but found 4         indent
  20:9   error  'location' is never reassigned. Use 'const' instead  prefer-const
  [21](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:22):1   error  Expected indentation of 2 spaces but found 4         indent
  [23](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:24):1   error  Expected indentation of 2 spaces but found 4         indent
  [24](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:25):1   error  Expected indentation of 4 spaces but found 8         indent
  24:13  error  'floor' is never reassigned. Use 'const' instead     prefer-const
  [25](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:26):1   error  Expected indentation of 4 spaces but found 8         indent
  25:13  error  'legs' is never reassigned. Use 'const' instead      prefer-const
  [26](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:27):1   error  Expected indentation of 4 spaces but found 8         indent
  26:13  error  'head' is never reassigned. Use 'const' instead      prefer-const
  [28](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:29):1   error  Expected indentation of 4 spaces but found 8         indent
  28:28  error  Missing semicolon                                    semi
  [30](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:31):1   error  Expected indentation of 4 spaces but found 8         indent
  30:62  error  Missing semicolon                                    semi
  [31](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:32):1   error  Expected indentation of 4 spaces but found 8         indent
  31:62  error  Missing semicolon                                    semi
  [32](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:33):1   error  Expected indentation of 4 spaces but found 8         indent
  32:62  error  Missing semicolon                                    semi
  [34](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:35):1   error  Expected indentation of 4 spaces but found 8         indent
  [36](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:37):1   error  Expected indentation of 6 spaces but found 12        indent
  [37](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:38):1   error  Expected indentation of 6 spaces but found 12        indent
  [38](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:39):1   error  Expected indentation of 4 spaces but found 8         indent
  [39](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:40):1   error  Expected indentation of 2 spaces but found 4         indent
  [40](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:41):1   error  Expected indentation of 2 spaces but found 4         indent
  40:65  error  Missing semicolon                                    semi
  [44](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:45):1   error  Expected indentation of 2 spaces but found 4         indent
  44:9   error  'levels' is never reassigned. Use 'const' instead    prefer-const
  [46](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:47):1   error  Expected indentation of 2 spaces but found 4         indent
  [47](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:48):1   error  Expected indentation of 4 spaces but found 8         indent
  [48](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:49):1   error  Expected indentation of 2 spaces but found 4         indent
  [50](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:51):1   error  Expected indentation of 2 spaces but found 4         indent

/home/runner/work/WorldEdit-BE/WorldEdit-BE/src/server/commands/navigation/descend.ts
   7:1   error  Expected indentation of 2 spaces but found 4         indent
   8:1   error  Expected indentation of 2 spaces but found 4         indent
   9:1   error  Expected indentation of 2 spaces but found 4         indent
  10:1   error  Expected indentation of 2 spaces but found 4         indent
  11:1   error  Expected indentation of 4 spaces but found 8         indent
  12:1   error  Expected indentation of 6 spaces but found 12        indent
  13:1   error  Expected indentation of 6 spaces but found 12        indent
  14:1   error  Expected indentation of 6 spaces but found 12        indent
  15:1   error  Expected indentation of 4 spaces but found 8         indent
  16:1   error  Expected indentation of 2 spaces but found 4         indent
  20:1   error  Expected indentation of 2 spaces but found 4         indent
  20:9   error  'location' is never reassigned. Use 'const' instead  prefer-const
  21:1   error  Expected indentation of 2 spaces but found 4         indent
  23:1   error  Expected indentation of 2 spaces but found 4         indent
  24:1   error  Expected indentation of 4 spaces but found 8         indent
  24:13  error  'floor' is never reassigned. Use 'const' instead     prefer-const
  25:1   error  Expected indentation of 4 spaces but found 8         indent
  25:13  error  'legs' is never reassigned. Use 'const' instead      prefer-const
  26:1   error  Expected indentation of 4 spaces but found 8         indent
  26:13  error  'head' is never reassigned. Use 'const' instead      prefer-const
  28:1   error  Expected indentation of 4 spaces but found 8         indent
  28:28  error  Missing semicolon                                    semi
  30:1   error  Expected indentation of 4 spaces but found 8         indent
  30:[62](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:63)  error  Missing semicolon                                    semi
  31:1   error  Expected indentation of 4 spaces but found 8         indent
  31:62  error  Missing semicolon                                    semi
  32:1   error  Expected indentation of 4 spaces but found 8         indent
  32:62  error  Missing semicolon                                    semi
  34:1   error  Expected indentation of 4 spaces but found 8         indent
  36:1   error  Expected indentation of 6 spaces but found 12        indent
  37:1   error  Expected indentation of 6 spaces but found 12        indent
  38:1   error  Expected indentation of 4 spaces but found 8         indent
  39:1   error  Expected indentation of 2 spaces but found 4         indent
  40:1   error  Expected indentation of 2 spaces but found 4         indent
  40:[66](https://github.com/SIsilicon/WorldEdit-BE/runs/8123310493?check_suite_focus=true#step:8:67)  error  Missing semicolon                                    semi
  44:1   error  Expected indentation of 2 spaces but found 4         indent
  44:9   error  'levels' is never reassigned. Use 'const' instead    prefer-const
  46:1   error  Expected indentation of 2 spaces but found 4         indent
  47:1   error  Expected indentation of 4 spaces but found 8         indent
  48:1   error  Expected indentation of 2 spaces but found 4         indent
  50:1   error  Expected indentation of 2 spaces but found 4         indent
```

**Comment by Fynn93** [comment]
Fixed (see commit [#dd36a62](https://github.com/SIsilicon/WorldEdit-BE/pull/112/commits/dd36a62ef04e8936a749bae760caa9d95fc36b87))

**Comment by SIsilicon** [comment]
Thanks!

**Pull request merged**